src/store/selectors/wallets.js

```diff
@@ -42,6 +42,11 @@ export function selectWalletsRenameAddress(state: AppState): WalletsRenameAddres
   return state.walletsRenameAddress
 }
 
+export function selectActiveWallet(state: AppState): ?Wallet {
+  const activeWalletId = selectActiveWalletId(state)
+  return selectWalletsItems(state).filter(wallet => wallet.id === activeWalletId)[0]
+}
+
```

Comment:
utils/getWallet exists to do the same :)

Reply:
Yes, but `getWallet` needs two parameters: items and activeWalletId, in this case, getting the two values by ​​selectors in `mapStateToProps` more difficult . I guess, better way is: take the data in selector and apply selector in `mapStateToProps`.

Reply:
agree, but please use Array.find